src/traversals/greedy_color.jl

```diff
@@ -39,28 +39,27 @@ function perm_greedy_color(g::AbstractGraph, seq::Vector{T}) where T <: Integer
         end
     end
 
-    return coloring{T}(maximum(cols), cols)
+    return Coloring{T}(maximum(cols), cols)
 end
 
 """
     degree_greedy_color(g)
 
 Color graph `g` iteratively in the descending order of the degree of the vertices.
 """
-function degree_greedy_color(g::AbstractGraph{T}) where T <: Integer 
+function degree_greedy_color(g::AbstractGraph{T}) where {T <: Integer} 
```

Comment:
Why the braces here? It's not consistent with the rest of the file.

Reply:
@sbromberger I added braces everywhere for consistency, I find it hard to parse when there is more than just a `T`